### DISCUSSION TITLE
大佬牛逼

### DISCUSSION THREAD

**Discussion by ThisIszas** [linearmouse/linearmouse#109] [General]
从scrollReverser过来的, 大佬体现了真正的开源精神

**Comment by lujjjh** [comment]
过誉了，很高兴我的修复可以帮到你。

Scroll Reverser 作为一款优秀的应用，对很多人来说刚好够用，相比 LinearMouse 有着更大的用户体量。所以，向 Scroll Reverser 提交代码是一件双赢的事情：一方面可以帮助更多人解决问题，另一方面也可以吸引更多人测试修复的方案是否有效。

---

Thanks for your kind words. I'm glad my fix helps.

Scroll Reverser is a great app, just perfect for many people, and has a much larger user base than LinearMouse. So submitting code to Scroll Reverser is a win-win: on the one hand, it helps more people solve their problems; on the other hand, it attracts more people to test whether the fixes work.
